_config/theme.yml

```diff
@@ -0,0 +1,12 @@
+
+---
+Name: accessway
```

Comment:
We're calling it `silverstripe-login-forms` - I think "accessway" is an internal name, not sure we should bake it into code